### ISSUE TITLE
Crash on Echoes Operation Stranglehold

### ISSUE THREAD

**Issue by n0samu** [ruffle-rs/ruffle#10423]
### Describe what you were doing

SWF URL: https://echoesgame.com/dl/echoes_act1_12_secure.swf

Steps to reproduce:
1. Click Play
2. Click Jump several times to skip through the cutscenes (if you don't want to sit through them)
3. Click Start Campaign
4. Click Play
5. Click Skip Scene (if you don't want to sit through the cutscene)
6. The game will crash

### What does the crash message say?

panicked at 'wgpu error: Validation Error

Caused by:
    In CommandEncoder::copy_texture_to_texture
    Copy error
    copy of X 233..483 would end up overrunning the bounds of the Destination texture of X size 250

', C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\wgpu-0.15.1\src\backend\direct.rs:3024:5

### Ruffle Version

0.1.0-nightly (fc4b51b 2023-03-27)

### Affected platform

Desktop app

### Operating system

Windows 10.0.19045 (Windows 10 Pro) [64-bit]

### Additional information

Filename: echoes_act1_12_secure.swf

### AVM2 Callstack
```
	at flash.display::BitmapData/flash::display::BitmapData::applyFilter()
	at _-NR/_-fr()
	at _-NR/_-K4()
	at flash.events::EventDispatcher/flash::events::EventDispatcher::dispatchEvent()
	at _-I7/_-8A()
```

According to @Lord-McSweeney on Discord, this might be an off-by-1 error (either by us or by wgpu):
> I think the issue with echo is an off-by-1
copy of X 233..483 would end up overrunning the bounds of the Destination texture of X size 250
483-233 = 250

**Comment by n0samu** [comment]
Note: This is **not** fixed by #10315 or #10254!

**Comment by n0samu** [comment]
This will be fixed by #10502 🎉
